draft-ietf-quic-qpack.md

```diff
@@ -715,6 +714,132 @@ None.
+| 67    | :status                          | 103                                                                                                    |
+| 68    | :status                          | 204                                                                                                    |
+| 69    | :status                          | 206                                                                                                    |
+| 70    | :status                          | 302                                                                                                    |
+| 71    | :status                          | 400                                                                                                    |
+| 72    | :status                          | 403                                                                                                    |
+| 73    | :status                          | 404                                                                                                    |
+| 74    | :status                          | 421                                                                                                    |
+| 75    | :status                          | 500                                                                                                    |
+| 76    | :status                          | 503                                                                                                    |
+| 77    | p3p                              |                                                                                                        |
+| 78    | content-encoding                 | gzip                                                                                                   |
+| 79    | content-encoding                 | br                                                                                                     |
+| 80    | accept-ranges                    | bytes                                                                                                  |
+| 81    | alt-svc                          | clear                                                                                                  |
+| 82    | x-xss-protection                 | 1; mode=block                                                                                          |
+| 83    | host                             |                                                                                                        |
+| 84    | pragma                           | no-cache                                                                                               |
+| 85    | pragma                           | public                                                                                                 |
```

Comment:
No.

Reply:
To `pragma` entirely, or this particular value?  It's outside spec, but it's commonly used.

Reply:
This particular value definitely, but probably 84 too. We shouldn't be encouraging HTTP/1.0 semantics and this isn't going to result in any real-world perceivable performance benefit.